src/app/models/api-models/application/application.ts

```diff
@@ -17,7 +17,7 @@ interface ApplicationApiAttributes {
   invoice: Invoice;
   job: Job;
   jobEnded: boolean;
-  language: Language;
+  language: Language; // TODO missing relation in docs?
```

Comment:
Not under relationships in the route docs. 😄 @buren The language_id is there though